src/main/java/com/futurecollars/FilesTask7/Main.java

```diff
@@ -0,0 +1,14 @@
+package com.futurecollars.FilesTask7;
+
+public class Main {
+
+    public static void main(String[] args) throws IllegalArgumentException {
```

Comment:
ten throws wydaje sie zbedny, IllegalArgumentException to chyba unchecked ;) https://howtodoinjava.com/java/exception-handling/checked-vs-unchecked-exceptions-in-java/#:~:text=Remember%20the%20biggest%20difference%20between,used%20to%20indicate%20programming%20errors.